ts/features/lollipop/store/reducers/lollipop.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * A reducer for lollipop.
+ */
+import { isActionOf } from "typesafe-actions";
+import { Action } from "../../../../store/actions/types";
+import { lollipopKeyTagSave } from "../actions/lollipop";
+import { GlobalState } from "../../../../store/reducers/types";
+
+export type LollipopState = Readonly<{
+  keyTag?: string;
+}>;
+
+export const initialLollipopState: LollipopState = {
+  keyTag: undefined
+};
+
+export default function lollipopReducer(
+  state: LollipopState = initialLollipopState,
+  action: Action
+): LollipopState {
+  if (isActionOf(lollipopKeyTagSave, action)) {
+    return {
+      ...state,
+      keyTag: action.payload.keyTag
+    };
+  }
+  return state;
+}
+
+export const lollipopSelector = (state: GlobalState) => state.lollipop;
```

Comment:
We can create another selector to pick `keyTag` directly.
what do you think?

Reply:
Why not! Good point!